src/runtime-rs/Cargo.lock

```diff
@@ -412,6 +412,18 @@ dependencies = [
  "thiserror",
 ]
 
+[[package]]
+name = "cgroups-rs"
+version = "0.2.11"
```

Comment:
But this version is not right here if you have reworked it already

Reply:
updated